src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,89 @@
+package frc.robot.subsystems.intake;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Intake extends LoggedSubsystem<IntakeLoggedInputs> {
+    public static Intake INSTANCE;
+    private final CANSparkMax higherSparkMax = new CANSparkMax(Ports.Intake.LOWER_SPARK_MAX_PORT, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax lowerSparkMax = new CANSparkMax(Ports.Intake.HIGHER_SPARK_MAX_PORT, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final Solenoid solenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Intake.SOLENOID);
+
+    private Intake() {
+        super(new IntakeLoggedInputs());
+        higherSparkMax.setIdleMode(CANSparkMax.IdleMode.kCoast);
+        lowerSparkMax.setIdleMode(CANSparkMax.IdleMode.kCoast);
+    }
```

Comment:
When using spark max you need to call the flush method after the configuration is done

Reply:
what is a flush method?